src/app/business/delfin/masking-views/masking-views.component.ts

```diff
@@ -0,0 +1,146 @@
+import { Component, OnInit, ViewContainerRef, ViewChild, Directive, ElementRef, HostBinding, HostListener, PipeTransform, Pipe, Input,  } from '@angular/core';
+import { DomSanitizer } from "@angular/platform-browser";
+import { Router, ActivatedRoute } from '@angular/router';
+import { I18NService, Utils, Consts} from 'app/shared/api';
+import { FormControl, FormGroup, FormBuilder, Validators, ValidatorFn, AbstractControl } from '@angular/forms';
+import { AppService } from 'app/app.service';
+import { I18nPluralPipe } from '@angular/common';
+import { trigger, state, style, transition, animate } from '@angular/animations';
+import { Message, MenuItem ,ConfirmationService} from '../../../components/common/api';
+import { DelfinService } from '../delfin.service';
+
+
+
+let _ = require("underscore");
+
+
+@Component({
+    selector: "app-delfin-masking-views",
+    templateUrl: "masking-views.component.html",
+    providers: [ConfirmationService],
+    styleUrls: [],
+    animations: [],
+})
+export class MaskingViewsComponent implements OnInit {
+    @Input() selectedStorage;
+
+    selectedStorageId: any;
+    showRightSidebar: boolean = false;
+    showDetails: boolean = false;
+    selectedMaskingView: any;
+    allMaskingViews: any;
+    msgs: Message[];
+    detailsLabels = {
+        heading: ""
+    }
+    detailSource: any;
+    detailsObj: any;
+    label = {
+        name: "Name",
+        description: "Description",
+        createdAt: "Created At",
+        updatedAt: "Updated At",
+        hostName: "Name",
+        ip: "IP Address",
+        port: "Port",
+        status: "Status",
+        osType: "OS",
+        accessMode: "Access Mode",
```

Comment:
Are these elements needed (accessMode, availabilityZones)?

Reply:
Not needed but these are labels used for display of Hosts from our On premise service. 
These can be removed.